Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -1609,7 +1612,7 @@ void WebProcess::prepareToSuspend(bool isSuspensionImminent, MonotonicTime estim
         platformMediaSessionManager->processWillSuspend();
 #endif
 
-#if !PLATFORM(MAC)
+#if !PLATFORM(MAC) && !PLATFORM(IOS)
```

Comment:
Doesn't this disable process-slimming on iOS before suspension? This doesn't seem like an acceptable / shippable change as it would result in more background WebContent jetsams, which would be a net user regression.